tfe/resource_tfe_workspace.go

```diff
@@ -54,16 +62,24 @@ func resourceTFEWorkspace() *schema.Resource {
 				Default:  false,
 			},
 
+			"execution_mode": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ConflictsWith: []string{"operations"},
+			},
```

Comment:
missed this before but we should probably validate the execution mode values. it'd be better if we'd stored these as constants in go-tfe but we can always fix that later
something like this should work:
```suggestion
			"execution_mode": {
				Type:          schema.TypeString,
				Optional:      true,
				Computed:      true,
				ConflictsWith: []string{"operations"},
				ValidateFunc: validation.StringInSlice(
					[]string{
						"agent",
						"local",
						"remote",
					},
					false,
				),
			},
```